server/src/routines/turn-actions.ts

```diff
@@ -162,6 +162,11 @@ export function playCardAction(
 		'You can not play a card in a slot with a card in it',
 	)
 
+	assert(
+		!game.components.filter(SlotComponent, query.slot.frozen).includes(pickedSlot),
+		'You cannot interact with cards in frozen slots'
```

Comment:
```suggestion
		!query.slot.frozen(game, card.slot),
		'You cannot play cards that are in frozen slots'
```
Is this your intention?

Reply:
I think so.